packages/fxa-settings/src/styles/header-content.css

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I am not sure that we need this CSS file. Does it make sense to apply these classes directly instead to [avoid premature abstraction](https://tailwindcss.com/docs/reusing-styles#avoiding-premature-abstraction)?

Reply:
Maybe I misunderstood what you meant here:

> Also, header should have a class of `card-header` similar to what's in content-server now.

There's so few styles applied per element that I do feel it would be cleaner to apply the classes directly.

Reply:
I should have been more explicit - a `card-header` class already exists in `flows.css` in `fxa-react`. This was created in #13739 when we'd started converting a lot of pages over to Tailwind, so there are pages that can be referenced/inspected to help with styling (anything without `#main-content`). You should be able to just use that class and `card-subheader` without redefining them here.

<img width="1316" alt="image" src="https://user-images.githubusercontent.com/13018240/207751999-0eec76fc-3809-411e-9bbe-4908c480102a.png">

Reply:
Now that you bring that up, I feel like `CardHeader` might be a better component name. 

The styles might be more accurate if named `card-heading` and `card-subheading` since the styles are applied to h1 vs header element  🤔 but I think I'll skip making a change that will ripple out on other components in settings and payments.